### ISSUE TITLE
Install on a hosted service

### ISSUE THREAD

**Issue by ThorsenRune** [BookStackApp/BookStack#2549]
Hi, will it be possible to install locally and then just upload the files using filezilla to a php server  (plesk) and run it?
I have tried but it just gives me error 500
Thanks

**Comment by ssddanbrown** [comment]
There are ways to do it, but it's not ideal nor something we support. It's generally easy to go wrong when going down this path and ending up with an insecure setup.

If possible ensure you're only serving the `public` folder. Many hosts will let you change the web-root or host-folder or something similar, this will need to be the `public` folder.

**Comment by ThorsenRune** [comment]
Managed to do it.
My webhost  (hostingperte) does not allow issuing commands (Its not possible to  use ssh on the server)

Therefore the installation described on bookstackapp is not immediately possible.

On windows install XAMPP and Composer-setup.exe

see: https://github.com/BookStackApp/BookStack/issues/2551

I have made a local installation with the name (DB_DATABASE) and credentials (DB_USERNAME, DB_PASSWORD) of the database on HostingperTe.

Then I uploaded the thing using filezilla.

Ran the local version, setting following the installation 

Dumped the local table

Uploaded it via MyPhpAdmin to the server

changed the 

APP_URL to my sitename in the .env file.

Then uploaded .env file and it seems to work
